AutomatedTesting/Gem/PythonTests/editor/EditorScripts/Sample_Editor_Tests.py

```diff
@@ -0,0 +1,143 @@
+    found_component_typeId = ("Found component typeId", "Unable to find component TypeId")
+    remove_mesh_component = ("Mesh component removed successfully", "Failed to remove mesh component")
+
+
+def sample_editor_tests():
+    """
+        Performing basic test in editor
+            01. create_level if it does not exist else 02. open exiting level
+
+            03. create parent entity and set name
+            04. create child entity
+            05. delete child entity
+            06. add mesh component
+            07. remove mesh component
+            08. enter game mode
+            09. exit game_mode
+            10. delete parent entity
+            11. save level
+
+    """
+    import os
+    from editor_python_test_tools.utils import Report
+    from editor_python_test_tools.utils import TestHelper as helper
+    import editor_python_test_tools.hydra_editor_utils as hydra
+
+    import azlmbr.math as math
+    import azlmbr.asset as asset
+    import azlmbr.bus as bus
+    import azlmbr.editor as editor
+    import azlmbr.entity as entity
+    import azlmbr.legacy.general as general
+    import azlmbr.object
+
+    def search_entity(entity_to_search, entity_name):
+        """
+
+        :param entity_to_search: entity to be searched
+        :param entity_name: name of the entity used in the set command
+        :return: True if entity id exists in the entity_list
+                 False if entity id does not exist in the entity_list
+        """
+        entity_list = []
+        entity_search_filter = entity.SearchFilter()
+        entity_search_filter.names = entity_name
+        entity_list = entity.SearchBus(bus.Broadcast, 'SearchEntities', entity_search_filter)
+        if entity_list:
+            if entity_to_search in entity_list:
+                return True
+            return False
+        return False
+
+    # 01. create_level
+
+    test_level = 'Simple'
+    general.open_level_no_prompt(test_level)
+    Report.result(Tests.create_level, general.get_current_level_name() == test_level)
+
+    # 02. load existing level - skipping this since this could alter existing level that other test depends on
+
+    # 03. create_entity and set name
+    # Delete any exiting entity and Create a new Entity at the root level
+    search_filter = azlmbr.entity.SearchFilter()
+    all_entities = entity.SearchBus(azlmbr.bus.Broadcast, "SearchEntities", search_filter)
+    editor.ToolsApplicationRequestBus(bus.Broadcast, "DeleteEntities", all_entities)
+    parent_entity = editor.ToolsApplicationRequestBus(bus.Broadcast, "CreateNewEntity", entity.EntityId())
+    Report.result(Tests.create_entity, parent_entity.IsValid())
+
+    # Setting a new name
+    parent_entity_name = "Parent_1"
+    editor.EditorEntityAPIBus(bus.Event, 'SetName', parent_entity, parent_entity_name)
+    Report.result(Tests.set_entity_name,
+                  editor.EditorEntityInfoRequestBus(bus.Event, 'GetName', parent_entity) == parent_entity_name)
```

Comment:
using editor_entity_utils this shortens to
```suggestion
                  parent_entity.get_name() == parent_entity_name)
```

Reply:
fixed